### PR TITLE
Release notes 8.0 rc2

### PR DESCRIPTION
**Pull request by hythloda**
[Here is the rendered file.
](https://builds.gradle.org/repository/download/Gradle_Release_Check_BuildDistributions/60368019:id/distributions/gradle-8.0-all.zip!/gradle-8.0-20230112155144%2B0000/docs/release-notes.html)
### Contributor Checklist
- [x] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [x] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [x] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [x] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [x] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [x] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [x] Ensure that tests pass sanity check: `./gradlew sanityCheck`
- [x] Ensure that tests pass locally: `./gradlew <changed-subproject>:quickTest`

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation

### CODE REVIEW

**Comment by hythloda** [comment]
@bot-gradle test BD

**Comment by hythloda** [comment]
@bot-gradle test BD

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -140,86 +144,58 @@ Starting with Gradle 8.0, it now uses the configured Java Toolchain, or Java 8 i
...
 
-In Gradle 8.0, tasks run in parallel from the first build when using the configuration cache.
-Gradle now loads the set of tasks from the cache entry after storing them on a cache miss.
-These tasks are isolated and can run in parallel. This is more  fine-grained than using the --parallel flag.
+#### Improved configuration cache for parallelism on the first run
 
-When the [configuration cache](userguide/configuration_cache.html) is enabled, and Gradle is able to locate a compatible configuration cache entry for the requested tasks, it loads the tasks to run from the cache entry and runs them as so-called 'isolated' tasks.
-Isolated tasks are able to run in parallel by default, subject to dependency constraints.
+Configuration cache now enables more fine-grained parallelism than using the --parallel flag. Starting in Gradle 8.0, tasks run in parallel from the first build when using the configuration cache. These tasks are isolated and can run in parallel.
```
Comment (ljacomet) [inline subprojects/docs/src/docs/release/notes.md:185]:
```suggestion
Configuration cache now enables more fine-grained parallelism than using the `--parallel` flag.
Starting in Gradle 8.0, tasks run in parallel from the first build when using the configuration cache. These tasks are isolated and can run in parallel. 
```

**Review by ljacomet — APPROVED** [review]
LGTM and I really like the highlight section!

💄 Having one sentence per line would really improve review / suggestions here

Minor formatting suggestion as well, see below

**Comment by hythloda** [comment]
@bot-gradle test and merge

**Review by ljacomet — APPROVED** [review]

**Pull request merged**